Fix requirements for emoji plugin

FedericoCozziVM opened the pull request:
I encountered some issues installing this plugin, about unsatisfied conditions and requisites, so I thought to update readme and plugin with non-default packages

FedericoCozziVM committed:
Fix requirements for emoji plugin

- Added `fuzzywuzzy` as python requirement
- Upgraded plugin patch version to 0.4.1
- Modified README.md including dependency to `xclip`

FedericoCozziVM commented:
@bergercookie it's me or the failure of the build pipeline is related to other plugins, untouched by this PR?
```
============================= test session starts ==============================
platform linux -- Python 3.6.7, pytest-7.0.1, pluggy-0.12.0
rootdir: /home/travis/build/bergercookie/awesome-albert-plugins
collected 6 items                                                              
plugins/gmaps/gmaps-cli/tests/test_gmaps_cli.py ....s                    [ 83%]
plugins/saxophone/test/test_links.py F                                   [100%]
=================================== FAILURES ===================================
```
Could you check my code and eventually merge it anyway? I'm new to PRs on github, so I don't know if you can :smile:

bergercookie commented on plugins/emoji/README.md:
```diff
@@ -25,7 +25,30 @@ On additional key presses it allows for fuzzy search on the labels of each emoji
-Install [em-keyboard](https://github.com/hugovk/em-keyboard) via `pip3`:
+### Requisites
+
+#### Exec requisites
```
Maybe `Prerequisites` instead ?
Same for the rest of the usages of the term in the README

bergercookie commented on plugins/emoji/__init__.py:
```diff
@@ -13,14 +13,14 @@
 
 __title__ = "Emoji picker"
-__version__ = "0.4.0"
+__version__ = "0.4.1"
```
Let's revert this - this is the albert version this plugin depends on and currently all plugins in this repo use `0.4.0. Was this required for the plugin to work on your end?

FedericoCozziVM replied:
Sorry my bad, I thought that the version was specific for the plugin

bergercookie commented on plugins/emoji/README.md:
````diff
@@ -25,7 +25,30 @@ On additional key presses it allows for fuzzy search on the labels of each emoji
+```bash
+sudo apt-get install xclip
+which xclip  # i.e. /usr/bin/xclip
+export PATH="$PATH:/usr/bin/xclip"
````
For this to work you'd have to write

```
export PATH="$PATH:/usr/bin/" # <-- directory that xclip is in
```

But then again `/usr/bin` is always in your PATH on Ubuntu (and most other distros)

Also, if `which xclip` does find the executable, then it's most likely already in your PATH (see `man xclip` for more).

I'd skip the bash snippet in this case and just stick to:

> `xclip`: you must have installed xclip; ensure that xlip path is in PATH environment variable.`

Users will probably know how to modify PATH if required

FedericoCozziVM replied:
You're right, I'll change that

bergercookie commented:
Hi @FedericoCozziVM , the CI failure is indeed irrelevant - don't worry about it.
Other than that left some minor comments - once addressed we can merge :)

FedericoCozziVM committed:
Requested changes on PR

- Reverted emoji plugin version to 0.4.0
- Changed "Requisites" to "Prerequisites" in emoji/README.md
- Simplified exec prerequisites section in emoji/README.md

bergercookie commented:
Thanks!
